packages/components/src/components/form/Textarea/Textarea.tsx

```diff
@@ -86,45 +83,40 @@ export const Textarea = ({
     };
 
     const formattedCharacterCount = getCharacterCount();
-    const isWithLabel = label || labelHoverAddon || labelRight;
 
     return (
         <Wrapper
             className={className}
-            {...wrapperProps}
             onMouseEnter={() => setIsHovered(true)}
             onMouseLeave={() => setIsHovered(false)}
+            data-test={dataTest}
         >
-            {isWithLabel && (
-                <Label>
-                    <LabelLeft>{label}</LabelLeft>
-                    <LabelRight>
-                        {labelHoverAddon && (
-                            <LabelHoverAddon isVisible={isHovered}>
-                                {labelHoverAddon}
-                            </LabelHoverAddon>
-                        )}
-                        {labelRight && <RightLabel>{labelRight}</RightLabel>}
-                    </LabelRight>
-                </Label>
-            )}
-
-            <StyledTextarea
-                spellCheck={false}
-                autoCorrect="off"
-                autoCapitalize="off"
-                maxLength={maxLength}
-                disabled={isDisabled}
-                width={width}
-                inputState={inputState}
-                rows={rows}
-                ref={innerRef}
-                {...rest}
+            <TopLabels
+                isHovered={isHovered}
+                labelHoverAddon={labelHoverAddon}
+                labelRight={labelRight}
             />
 
-            {formattedCharacterCount && <CharacterCount>{formattedCharacterCount}</CharacterCount>}
+            <InputWrapper>
+                <StyledTextarea
+                    spellCheck={false}
+                    autoCorrect="off"
+                    autoCapitalize="off"
+                    maxLength={maxLength}
+                    disabled={isDisabled}
+                    inputState={inputState}
+                    rows={rows}
+                    placeholder={placeholder || ''} // needed for uncontrolled inputs
+                    ref={innerRef}
+                    {...rest}
+                />
+
+                {formattedCharacterCount && (
+                    <CharacterCount>{formattedCharacterCount}</CharacterCount>
+                )}
 
-            {children}
+                {label && <TextareaLabel isDisabled={isDisabled}>{label}</TextareaLabel>}
+            </InputWrapper>
```

Comment:
d083e0d435

Reply:
Works well although I like this approach a bit better: https://stackoverflow.com/a/72808802

`padding-right` on `CharacterCount` could be increased a little so that it does not overlap the scrollbar at all.

We could take a look at styling scrollbars in the future, too, the contrast is quite high with the dark theme.

Reply:
I wanted to preserve the common styles as much as possible :) The border top kinda breaks this. But the solutions I have is for sure is not the cleanest too.

oh, I don't have this wide scrollbar, missed it. Not sure styling them is the best idea though. 

046bc45e19